stories/object-list/object-list.stories.js

```diff
@@ -0,0 +1,1193 @@
+         style="background-image: url('http://lorempixel.com/400/400/nature/4/')">
+        </span>
+        <div class="fd-object-list__header-left">
+         <div class="fd-object-identifier fd-object-list__object-identifier">
+          <p class="fd-object-identifier__title">
+           Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+          </p>
+         </div>
+        </div>
+        <div class="fd-object-list__header-right">
+          <span class="fd-object-number fd-object-number--critical fd-object-list__object-number">
+            <span class="fd-object-number__text">457.00</span>
+            <span class="fd-object-number__unit">Euro</span>
+          </span>
+        </div>
+      </div>
+      <div class="fd-object-list__content">
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              
+              First Attribute
+            </div>
+          </div>
+          <div class="fd-object-list__row-right">
+            <div class="fd-object-marker">
+              <i class="fd-object-marker__icon sap-icon--user-edit" role="presentation"></i>
+                <span class="fd-object-marker__text">Draft</span>
+            </div>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Second Attribute
+            </div>
+          </div>
+        </div>
+        <div class="fd-object-list__row">
+          <div class="fd-object-list__row-left">
+            <div class="fd-object-list__object-attribute">
+              Third Attribute
+            </div>
+        </div>
+      </div>
+    </div>
+    </div>
+  </li>
+</ul>
+`;
+
+object.storyName = 'Object';
+
+object.parameters = {
+    docs: {
+        iframeHeight: 785,
+        storyDescription: ` 
+        Object number, Object status, Object identifier elements classes can be used to show there respective behaviours for examples
+        we can add these modifier classes to the Object number and object status elements to get the semantic status:
+      
+| Status | Object Number |   Object Status |
+| ----------------: | :------------: | :------------ |
+| Positive | \`fd-object-number--positive\` | \`fd-object-status--positive\` |
+| Negative | \`fd-object-number--negative\` | \`fd-object-status--negative\` |
+| Critical | \`fd-object-number--critical\` | \`fd-object-status--critical\` |
+| Informative | \`fd-object-number--informative\` | \`fd-object-status--informative\` |
+
+`
+    }
+};
+
+export const navigation = () => `
+
+<h4 id="objectListItemNavigation">Object List Item With Navigation</h4>
+
+<div style="max-width: 450px">
+<ul class="fd-list fd-object-list fd-list--navigation-object fd-list--navigation" role="listbox" aria-labelledby="objectListItemNavigation">
```

Comment:
Navigation items need to be wrapped in the navigation role with a valid label. See https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html 

```suggestion
<div role="navigation"  aria-labelledby="objectListItemNavigation">
<h4 id="objectListItemNavigation">Object List Item With Navigation</h4>
<ul class="fd-list fd-object-list fd-list--navigation-object fd-list--navigation">
```